src/python/turicreate/toolkits/one_shot_object_detector/util/_augmentation.py

```diff
@@ -52,6 +54,7 @@ def preview_synthetic_training_data(data,
     # {'mlmodel_path':'darknet.mlmodel', 'max_iterations' : 25}
     options_for_augmentation = {
         "seed": seed,
+        "user_provided": user_provided,
```

Comment:
Instead of adding a user option just for this purpose, wouldn’t it be simpler just to resize the images on the Python side?

Reply:
Fixed.